_sass/sidebar.scss

```diff
@@ -0,0 +1,29 @@
+.sidebar-left {
+  ul {
+    font-size:90%;
```

Comment:
There should be no font-size: percentage nowhere!
There are only several proper font-sizes, and no magic numbers or percentages are correct.